kpi/008.adoc

```diff
@@ -1,14 +1,17 @@
-== J. Links health
+== KPI-{counter:kpi}: Links health
 
 === Status
 
 Approved
 
 === WCMP element(s)
 
-Any element or attribute content with linked information (URLs). Links are
-generally present in `gmd:URL`, `gmd:graphicOverview`, `gmx:Anchor/@xlink:href`,
-and `@codeList` elements/attributes.
+Any element or attribute content with linked information (URLs).
+
+* `//gmd:URL`
+* `//gmd:graphicOverview`
+* `//gmx:Anchor/@xlink:href`
+* `//@codeList` TODO: should we really check code lists link health or treat as a plain old value?
```

Comment:
@tomkralidis for discussion,
 I would check

Reply:
I think we should at least test for valid URLs....

Reply:
TT-WISMD meeting 2021-02-19: As discussed, we will check for HTTP 200 and not inspect content.